change/@react-native-windows-cli-2020-11-12-23-06-58-nuget-0.63.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "port nuget 5.8 fix to 0.63",
```

Comment:
```suggestion
  "comment": "port nuget 5.8 fix and node 14 fix to 0.63",
```

Reply:
ah I think I got my merges wrong :-P the node 14 change went in separately